graphene_django/templates/graphene/graphiql.html

```diff
@@ -45,6 +45,9 @@
     {% if subscription_path %}
       subscriptionPath: "{{subscription_path}}",
     {% endif %}
+    {% if header_editor_enabled %}
+      graphiqlHeaderEditorEnabled: {{ graphiql_header_editor_enabled}},
+    {% endif %}
```

Comment:
```suggestion
      graphiqlHeaderEditorEnabled: {{ graphiql_header_editor_enabled|yesno:"true,false" }},
```

`header_editor_enabled` is not defined and the python boolean type needs to be converted to the js type.